cmake/Findelfutils.cmake

```diff
@@ -9,17 +9,17 @@
 # Lzma / ZLib Force static library by overriding CMAKE_FIND_LIBRARY_SUFFIXES since
 # FindZLIB/FindLIBLZMA don't provide the option
 set(ORIG_CMAKE_FIND_LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
-set(CMAKE_FIND_LIBRARY_SUFFIXES ".a")
+set(CMAKE_FIND_LIBRARY_SUFFIXES ".a" ".so")
```

Comment:
Why change this ?

Reply:
This is a mistake. Thanks for spotting this. There was an issue previously with lzma